docs: guidelines and add to readme (#1)
IBM/ibm-watson-embed-charts: README.md

# ibm-watson-embed-charts

Helm charts for the IBM Watson Libraries for Embed products.

> IBM Watson Libraries is designed to help lower the barrier for AI adoption by helping address the skills shortage and development costs that are required to build AI models from scratch.

Reviewer: ```suggestion
The IBM Watson Libraries for Embed products are designed to help lower the barrier for AI adoption by helping address the skills shortage and development costs that are required to build AI models from scratch.
```

## Documentation

> Find out more about the IBM Watson Libraries for Embed products  by reading the [product documentation](https://www.ibm.com/docs/watson-libraries).

Reviewer: The tiniest of NITs:
```suggestion
Find out more about the IBM Watson Libraries for Embed products by reading the [product documentation](https://www.ibm.com/docs/watson-libraries).
```

To learn more about embedding the products and working with the IBM sales team, visit the [Partner World page](https://www.ibm.com/partnerworld/program/embeddableai).

## License

The Helm charts are licensed under Apache 2.0.

> The containers the chart deploys are licensed under separate product licenses.

Reviewer: ```suggestion
The containers the charts deploy are licensed under separate product licenses.
```

Product licenses are included in the charts under the `licenses` directory.

> To run the containers and deploy the helm charts, the licenses must be accepted by setting up an override yaml configuration with `license: true`.

Reviewer: ```suggestion
To run the containers and deploy the helm charts, the licenses must be accepted by setting the chart value `license` to `true`.
```
